.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  publish_gh_pages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: main
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14"
+
+      - name: reset gh-pages to main
+        run: |
+          git checkout gh-pages
+          git reset --hard main
+
+      - name: install dependencies
+        run: npm ci
+        env:
+          PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: "1"
+
+      - name: build
+        run: |
+          npm run build
+
+      - name: publish to npm
+        run: npm publish
```

Comment:
What do we think about setting version to “0.0.0-git-tag” in main?
And then sed replacing it to git ref name (tag) before publish?

Reply:
do we need to commit to main?

would this work:

`main` will permanent have a package.json with “0.0.0-git-tag”
`gh-pages` will be main + build files + package.json with git ref name (tag) - and this is what we publish?

Reply:
like this:
<img width="648" alt="image" src="https://user-images.githubusercontent.com/33021497/179003458-e0260637-75c1-4c2e-b397-12c69e61e080.png">

it's weird.

Reply:
oh yea, server needs to be just `1.5.0` (no `v`) 

and the `0.0.0-git-tag` is supposed to mean "not versioned here, check the git tag"

so we wouldn't have a `0.0.0` tag

once this is working, we'll tag `1.5.0` and it'll publish that

Reply:
Oh, the `0.0.0` tag in the picture is just for illustration , not a real tag.

Reply:
> `main` will permanent have a package.json with “0.0.0-git-tag”
> `gh-pages` will be main + build files + package.json with git ref name (tag) - and this is what we publish?

Hmm, make senses.

Reply:
are you happy to make these changes? im also okay to merge this PR, and then i can work on the changes, just not 100% when ill get time

Reply:
I will make these changes later.😄

Reply:
Done